pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminApiTest2.java

```diff
@@ -1343,6 +1345,20 @@ public void testInvalidBundleErrorResponse() throws Exception {
         }
     }
 
+    @Test
+    public void testDedupTopicOwnership() throws Exception {
```

Comment:
Seems the tests can't cover the changes? I think we must start more than 1 brokers.